docs/pages/eas-update/expo-dev-client.mdx

```diff
@@ -8,67 +8,91 @@ import { Step } from '~/ui/components/Step';
+- Make sure your development build has the [`expo-updates` library installed](/eas-update/getting-started/#configure-your-project).
+
+## What is an Extensions tab
+
+<ImageSpotlight
+  alt="Extensions tab in a development build."
+  src="/static/images/eas-update/extensions-01.png"
+  style={{ maxWidth: 320 }}
+/>
+
+When using the `expo-updates` library inside a development build, the **Extensions** tab provides the ability to load and preview a published update automatically.
+
+### Preview an update using the Extensions tab
 
 <Step label="1">
 
-Create a [development build](/develop/development-builds/create-a-build) of the project.
+Make non-native changes locally in your project and then [publish them using `eas update`](/eas-update/publish/). The branch specified when running this command should correspond to a channel. You can run `eas channel:list` to view the channels linked to branches.
 
 </Step>
 
 <Step label="2">
 
-Make non-native changes locally, then publish them using `eas update --branch ...`. The branch specified should correspond to a channel. You can see how channels are linked to branches with `eas channel:list`.
+After publishing the update, open your development build, go to **Extensions**, and tap **Login** to log in to your Expo account within the development build. This step is required for the **Extensions** tab to load any published updates associated with the project under your Expo account.
 
 </Step>
 
 <Step label="3">
 
-After publishing an update, it's time to load the update in the development build. All development builds that have the `expo-updates` package installed include an Extensions tab where you can select an update:
+After logging in, an EAS Update section will appear inside the **Extensions** tab with one or more of the latest published updates. Tap **Open** next to the update you want to previous.
+
+</Step>
+
+In the **Extensions** tab, you can view the list of all published updates for a branch. Tap the branch name in the **Extensions** tab.
 
 <ImageSpotlight
-  alt="Extensions tab in the Expo dev client."
-  src="/static/images/dev-client/extensions-panel.png"
-  style={{ maxWidth: 300 }}
+  alt="Extensions tab in a development build."
+  src="/static/images/eas-update/extensions-02.png"
+  style={{ maxWidth: 480 }}
 />
 
-Alternatively, you can construct a specific URL to open your EAS Update. The URL will look like this:
+## Construct an update URL
+
+As an alternative to using the **Extensions** tab, you can construct a specific URL to open your EAS Update in the development build. The URL will look like the following:
 
 ```bash
 exp+[project-slug]://expo-development-client/?url=[https://u.expo.dev/project-id]?channel-name=[channel]
 
 # Example
-exp+form-duo://expo-development-client/?url=https://u.expo.dev/675cb1f0-fa3c-11e8-ac99-6374d9643cb2?channel-name=preview
+exp+my-app://expo-development-client/?url=https://u.expo.dev/675cb1f0-fa3c-11e8-ac99-6374d9643cb2?channel-name=preview
```

Comment:
This example will construct a URL to show the latest update mapped by the `preview` channel. There's a bunch of different usecases you can build based on our [www routes](https://github.com/expo/universe/blob/304feb31fd807b848f3fbf71ae42bfeb373dbc55/server/www/src/middleware/updateRoutes.ts#L30-L37). Might also be worth saying you can get the update preview from the website like [here ](https://expo.dev/accounts/quinlanj/projects/sentry-rn-50/updates/3d806a10-50a5-4abc-b821-6f32e4bd8af7).  @jonsamp is currently working to revamp that page, where you have a preview QR for a single update group instead of the platform-specific QR we have right now. So we are eventually going to show something like `exp+my-slug://u.expo.dev/:appId/group/:groupId`. For context, the update group id can be obtained from the website here: 
![Screenshot 2024-03-21 at 1 04 49 PM](https://github.com/expo/expo/assets/6380927/c1c38945-2a4e-44d1-b7e0-1334fc49764e)

Reply:
> This example will construct a URL to show the latest update mapped by the preview channel.

Thanks for catching that! I've fixed it by changing it to `development`. In the table below, we're also explaining that the channel name is `development`.

> here's a bunch of different usecases you can build based on our www routes.

Should we explain those use cases here?

> Might also be worth saying you can get the update preview from the website like here .

Since we do not cover those steps, I've added a section for this:

![CleanShot 2024-03-26 at 20 55 59@2x](https://github.com/expo/expo/assets/10234615/966b6260-17bd-4da4-b1b0-e7797aea5a9b)

> @jonsamp is currently working to revamp that page, where we'll want to eventually show you a preview QR for a single update group instead of the platform-specific QR we have right now. So we are eventually going to show something like exp+my-slug://u.expo.dev/:appId/group/:groupId. For context, the update group id can be obtained from the website here:

Shall I wait for Jon to revamp this page to add those instructions on this page? I can create a follow up task for this.

Reply:
> Should we explain those use cases here?
No need, just wanted to float the www routes as an fyi (very few people know of it's existence)!

Your revisions look good, thank you!